pyproject.toml

```
@@ -82,6 +82,7 @@ flake8-print = "^5.0.0"
 flake8-pytest-style = "^1.7.2"
 isort = "^5.13.2"
 jinja2-cli = {version = "==0.8.2", extras = ["yaml"]}
+moto="*"
```

Now you got me thinking if the API should just be referencing `*` for the version too...

In any case, just a quick formatting nitpick:

```suggestion
moto = "*"
```